sql/sql_yacc.yy

```diff
@@ -6767,8 +6773,8 @@ field_spec:
             LEX *lex=Lex;
             lex->parsing_options.lookup_keywords_after_qualifier= false;
             $$= $<create_field>2;
-
-            $$->check_constraint= $4;
+            if ($4)
+              $$->check_constraint= $4;
```

Comment:
what does that change?

Reply:
Removed, had problem with virtual columns from what I can remember.